[New] energy tutorial (#2452)
timescale/docs: tutorials/energy-data/index.md

granularity of a few seconds is key to finding all the answers for more
precision. This tutorial uses energy consumption data from a typical household
for over a year. Because nearly all of this data is time-series data, proper
analysis requires a purpose-built time-series database, like Timescale.

In this tutorial you can construct queries that look at how many watts were
consumed, and when. Additionally, you can visualize the energy consumption data
in Grafana.

## Prerequisites

Before you begin, make sure you have:

*   Signed up for a [free Timescale account][cloud-install].
*   <Optional />Signed up for a Grafana account][grafana-setup] to graph queries.

## Steps in this tutorial

This tutorial covers:

1.  [Setting up up your dataset][dataset-energy]: Set up and connect to a
    Timescale service, and load data into your database using `psql`.
1.  [Querying your dataset][query-energy]: Analyze a dataset containing energy
    consumption data using Timescale and PostgreSQL, and visualize the
    results in Grafana.

## About querying data with Timescale

This tutorial uses sample energy consumption data to show you how to construct

> queries for geospatial time-series data. The analysis you do in this tutorial is

Reviewer: ```suggestion
queries for time-series data. The analysis you do in this tutorial is
```

> similar to the kind of analysis civic organizations use to do things like plan
> new roads and public services.

Reviewer: ```suggestion
similar to the kind of analysis households might use to do things like plan
their solar installation, or optimize their energy use over time.
```

Or whatever you think ...

It starts by teaching you how to set up and connect to a Timescale database,
create tables, and load data into the tables using `psql`.

> You then learn how to conduct analysis and monitoring on your dataset. It walks

Reviewer: ```suggestion
You then learn how to conduct analysis and monitoring on your dataset. It also walks
```
